src/controllers/shop/checkout-item.controller.js

```diff
@@ -10,14 +10,23 @@ const checkoutItem = async (req, res) => {
       attributes: { exclude: ["password"] },
     });
 
-    const item = req.body;
+    const items = req.body;
 
     const transactions = [];
 
-    //for (const item of items) {
-      const { product_id, qty } = item;
+    for (const item of items) {
+      const { cart_id, qty } = item;
```

Comment:
@justJooo disini ku mau benerin karena kalau misalnya berdasarkan `cart_id` ngga perlu `qty` lagi karena di keranjang udah didefinisikan buat quantitynya contohnya seperti ini:

![image](https://github.com/Belega-Village-UNUD/backend-api-edb/assets/75294452/2b414dae-ec98-4037-91f5-d3a1e1b2c92f)

jadi request bodynya akan diubah jadi `cart_id` saja

Reply:
refer to commit : https://github.com/Belega-Village-UNUD/backend-api-edb/pull/51/commits/63b0a8f98a2e368a94cc9eef20db18b9d6fc6ac6